redfish-core/include/registries/openbmc_message_registry.hpp

```diff
@@ -241,6 +241,16 @@ constexpr std::array registry =
             },
             "None.",
         }},
+    MessageEntry{
```

Comment:
Need to actually make the change in openbmc.json in the same directory and then build the openbmc_message_registry.hpp from that.

(See L2-L10 of this file.)

Also, the subminor version is supposed to be updated when a new message is added.

Reply:
done, and bumped the registry subminor version to 0.6